src/main/java/org/opentripplanner/street/model/edge/StreetVehicleRentalLink.java

```diff
@@ -2,8 +2,8 @@
 
 import org.locationtech.jts.geom.LineString;
 import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.service.vehiclerental.street.VehicleRentalPlaceVertex;
```

Comment:
This class should be moved to `org.opentripplanner.service.vehiclerental.street`